### PR TITLE
add query channels

### PR DESCRIPTION
**Pull request by edjroz**

### CODE REVIEW

File: examples/hyperspace/hyperspace_test.go [hunk 1]
```diff
@@ -264,27 +268,38 @@ func TestHyperspace(t *testing.T) {
...
+	require.Equal(t, polkadotChannelOutput[0].PortID, "transfer")
+
+	require.Equal(t, len(polkadotChannelOutput), 1)
+
 	//fmt.Println("Polkadot connection: ", polkadotConnections[0].ID)
-	//err = testutil.WaitForBlocks(ctx, 2, polkadotChain, cosmosChain)
-	//require.NoError(t, err)
-	
+	err = testutil.WaitForBlocks(ctx, 2, polkadotChain, cosmosChain)
+	require.NoError(t, err)
```
Comment (misko9) [inline examples/hyperspace/hyperspace_test.go:301]: I think the WaitForBlocks can be removed

File: examples/hyperspace/hyperspace_test.go [hunk 2]
```diff
@@ -174,7 +173,12 @@ func TestHyperspace(t *testing.T) {
 	fmt.Println("About to create interchain")
 	ic := ibctest.NewInterchain().
 		AddChain(polkadotChain).
-		AddChain(cosmosChain).
+		AddChain(cosmosChain, ibc.WalletAmount{
+			// Use test keys temporarily
+			Address: "cosmos1nnypkcfrvu3e9dhzeggpn4kh622l4cq7wwwrn0",
+			Denom:   "stake",
+			Amount:  10_000_000_000_000,
+		}).
```
Comment (misko9) [inline examples/hyperspace/hyperspace_test.go:181]: Does this wallet need to be re-added? It shouldn't be needed anymore.

File: examples/hyperspace/hyperspace_test.go [hunk 3]
```diff
@@ -264,27 +268,38 @@ func TestHyperspace(t *testing.T) {
...
 	require.NoError(t, err)
 
 	r.(*hyperspace.HyperspaceRelayer).DockerRelayer.PrintConfigs(ctx, eRep, cosmosChain.Config().ChainID)
 	r.(*hyperspace.HyperspaceRelayer).DockerRelayer.PrintConfigs(ctx, eRep, polkadotChain.Config().ChainID)
-	
+
 	// Hyperspace panics on "hyperspace query channels --config xxx", this is needed.
-	//_, err = r.GetChannels(ctx, eRep, cosmosChain.Config().ChainID)
-	//require.NoError(t, err)
+	cosmosChannelOutput, err := r.GetChannels(ctx, eRep, cosmosChain.Config().ChainID)
```
Comment (misko9) [inline examples/hyperspace/hyperspace_test.go:283]: Can you replace any hardcoded channels and ports with this new info below?

File: relayer/hyperspace/hyperspace.go [hunk 4]
```diff
@@ -340,27 +338,27 @@ func (hyperspaceCommander) GeneratePath(srcChainID, dstChainID, pathName, homeDi
...
 	panic("[UpdatePath] Do not call me")
-	
+
 }
 
 func (hyperspaceCommander) GetChannels(chainID, homeDir string) []string {
-	panic("Panic because hyperspace will panic")
-	/*fmt.Println("[hyperspace] Get Channels")
-	configFilePath := path.Join(homeDir, chainID + ".config")
+	//panic("Panic because hyperspace will panic")
```
Comment (misko9) [inline relayer/hyperspace/hyperspace.go:345]: Can just remove the panic instead of commenting it

Comment (edjroz) [reply relayer/hyperspace/hyperspace.go:345]: Will do 👍

File: examples/hyperspace/hyperspace_test.go [hunk 5]
```diff
@@ -400,7 +395,7 @@ func TestHyperspace(t *testing.T) {
 	require.Equal(t, pollForBalance.Amount, cosmosUserNativeBal)
 	fmt.Println("Initial: ", cosmosUserAmount, "   Middle:", cosmosUserBalNew, "  Final: ", cosmosUserNativeBal)
 	// Trace IBC Denom
-	srcDenomTrace2 := transfertypes.ParseDenomTrace(transfertypes.GetPrefixedDenom("transfer", "channel-0", "UNIT"))
+	srcDenomTrace2 := transfertypes.ParseDenomTrace(transfertypes.GetPrefixedDenom(cosmosChannelOutput[0].PortID, cosmosChannelOutput[0].ChannelID, "UNIT"))
```
Comment (misko9) [inline examples/hyperspace/hyperspace_test.go:398]: These should be from polkadotChannelOutput

**Review by misko9 — APPROVED** [review]

**Pull request merged**